recipes/libxlsxwriter/all/conanfile.py

```diff
@@ -30,16 +38,24 @@ def _source_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.settings.os != "Linux":
+            del self.options.fmemopen
 
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
+    def validate(self):
+        if tools.Version(self.version) <= "1.0.5" and self.options.md5 == "openssl":
+            raise ConanInvalidConfiguration("{0}:md5=openssl is not suppported in {0}/{1}".format(self.name, self.version))
+
     def requirements(self):
         self.requires("minizip/1.2.11")
         self.requires("zlib/1.2.11")
+        if self.options.md5 == "openssl":
+            self.requires("openssl/1.1.1c")
```

Comment:
```suggestion
            self.requires("openssl/1.1.1l")
```

Can you the latest patch version?

Reply:
I agree with you. 
There are no reason to use old version.
Now I understand version policy in conan.

Reply:
Thank you! Some few projects really a specific version due API compatibility, fortunately, we don't have such case here.

Reply:
Thank you for detail information.
I understand.